File: SecondSemester/EducationalPractice/lab2.java
import static java.lang.Math.abs;
import static java.lang.Math.pow;
import static java.lang.Math.sqrt;

import java.util.Scanner;

class lab2 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
		System.out.print("x: ");   
        double x = scan.nextDouble();
        System.out.print("y: ");
        double y = scan.nextDouble();
        scan.close();
        double a = 0;
        double b = 0;
        
        b = (sqrt(abs(x + y))) / pow(x, abs(y - 2));
        a = x > y ? b + pow(x, 2) : pow(y, 2) - b;

        System.out.println("a: " + a + " b: " + b);
    }
}

File: SecondSemester/EducationalPractice/lab2.py
from math import *

x = int(input("x: "))
y = int(input("y: "))

b = (sqrt(abs(x + y))) / (x ** abs(y - 2))

a = b + x ** 2 if x > y else y ** 2 - b

print("a: " + str(a) + ", b: " + str(b))